connector/src/test/scala/com/google/cloud/spark/bigquery/it/SparkBigQueryEndToEndReadITSuite.scala

```diff
@@ -163,6 +165,32 @@ class SparkBigQueryEndToEndReadITSuite extends FunSuite
 
   def testsWithReadInFormat(dataSourceFormat: String, dataFormat: String): Unit = {
 
+    test(("testing view with different columns for select and filter. " +
+      "DataSource %s. Data Format %s").format(dataSourceFormat, dataFormat)) {
+
+      for(i <- 1 to 2){
+        var df = spark.read.format(dataSourceFormat)
```

Comment:
please use `val` and add another value for the cached DF

Reply:
Done